pipelines/unified_quickstart/alt-2_calculate_avg_gpa_azure.wdl

```diff
@@ -0,0 +1,49 @@
+version 1.0
+
+workflow CalculateStudentGPA {
+    meta {
+        description: "Calculate the GPA (average) for a student."
+    }
+
+    input {
+        Float   math_score
+        Float   language_score
+        Float   science_score
+        Int     num_scores
+    }
+
+    call CalculateAverage {
+        input:
+            math_score      = math_score,
+            language_score  = language_score,
+            science_score   = science_score,
+            num_scores      = num_scores
+    }
+
+    output {
+        Float   gpa = CalculateAverage.average
+    }
+}
+
+task CalculateAverage {
+    input {
+        Float   math_score
+        Float   language_score
+        Float   science_score
+        Int     num_scores
+    }
+
+    command <<<
+        python -c "print((~{math_score} + ~{language_score} + ~{science_score})/~{num_scores})"
+
+    >>>
+    
+
+    runtime {
+        docker: "terradevacrpublic.azurecr.io/workshop-python:3.12-debian"
```

Comment:
Is there a job to rebuild this image regularly (daily, weekly)?

This is a requirement to pull in security updates that get published for the base image.

Reply:
Oops, I now see the answer is in the PR description; I think that unless we set up a job, regular old `python:slim` might still be a better choice for Azure.

Reply:
Alex Baumann had mentioned something around dockerhub not being thrilled with the amount of image bytes we pull. 
But I get the feedback and the potential security implications, which I think also applies the other images in our ACR registry? I'll check in with AppSec to see if automating image updates for ACR is in their radar.

Reply:
I'll leave it up to you. Regular security patching is probably not the most critical for a workshop. On the other hand, we should be demonstrating best practices to our users!

Reply:
I agree, it's important. I'll update to (a specific version of) `python:slim`

Reply:
Agreed with @aednichols

Reply:
That said, on GCP it's really faster to just Google Artifact Registry - and I believe there was some magic we/Google used where when requests would come out to Docker Hub for popular images, they would actually be routed through a mirror https://cloud.google.com/artifact-registry/docs/pull-cached-dockerhub-images